scripts/kubescape-backlog-bridge/main_test.go

```
@@ -0,0 +1,595 @@
+// The anti-churn guarantee: identical state must re-derive an identical
+// fingerprint. Input order is shuffled to prove the result does not depend on
+// map iteration or slice order.
+func TestFingerprintIsStableAcrossRunsAndInputOrder(t *testing.T) {
+	base := itemsOf(t,
+		postureDoc("app", "Deployment", "api", map[string]string{"C-0016": "failed"}),
+		postureDoc("app", "Deployment", "web", map[string]string{"C-0016": "failed"}),
+		postureDoc("data", "StatefulSet", "db", map[string]string{"C-0016": "failed"}),
+	)
+	want := mustDerivePosture(t, base, nil)[0].Fingerprint()
+
+	rng := rand.New(rand.NewSource(1))
```

golangci-lint (2.12.2)

[high] G404: Use of weak random number generator (math/rand or math/rand/v2 instead of crypto/rand)

(gosec)